build.gradle

```diff
@@ -17,17 +17,16 @@
 buildscript {
     repositories {
         maven { url "https://maven-central.storage.googleapis.com"}
-        maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.1.10'
         classpath "net.ltgt.gradle:gradle-errorprone-plugin:0.0.8"
```

Comment:
I don't think errorprone is coming from the gradle plugin repository (https://plugins.gradle.org/) in this case